Braintree Component/BraintreeComponents/BraintreeComponents.csproj

```diff
@@ -1,13 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <RootNamespace>Braintree_Components</RootNamespace>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Braintree" Version="5.9.0" />
     <PackageReference Include="DecisionsSDK" Version="7.6.1" />
```

Comment:
is this the DecisionsSDK version we want to target, or are we trying to bring it up to 8.x?

Reply:
Targeting 8.x SDK makes sense. I can work on that while I'm also updating the Braintree SDK